### ISSUE TITLE
Memory leak related to helpers when isCached = false

### ISSUE THREAD

**Issue by yuvke** [hapijs/vision#171]
When running vision with `isCached=false` and at least one helper in `helpersPath` a memory leak manifests itself.

This (for now) has forced us to stop working with `isCached=false` altogether. 

I've created a [small repo](https://github.com/yuvke/test-hapi-vision) to easily reproduce / debug the issue.
After downloading run
``` bash
npm i
npm run start-server
npm run start-monitor
npm run start-load-test // on a separate tab
```
Heap memory keeps climbing for each request.

**Comment by yuvke** [comment]
Added a pull request (https://github.com/hapijs/vision/pull/172) that fixes this issue.

**Comment by wswoodruff** [comment]
Fixed by #172
